foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/util/DataLineageUtil.java

```diff
@@ -0,0 +1,74 @@
+package com.boozallen.aissemble.upgrade.util;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class DataLineageUtil {
+
+    public static String VERSION_AISSEMBLE_POST_1_7_0 = "1.7.0";
+
+    public static final String VERSION_AISSEMBLE_POST_1_7_0_REGEX = "com\\.boozallen\\.aissemble\\.data\\.lineage";
+
+    public static final String VERSION_AISSEMBLE_PRE_1_7_0_REGEX = "com\\.boozallen\\.aiops\\.data\\.lineage";
+
+    public static final String EQUALS = "=";
+
+    public static void replaceTextInFile(File file, String textToReplace, String replacementText) {
+        Path path = file.toPath();
+
+        System.out.println("Text to Replace: " + textToReplace +"Replacement Text: " + replacementText);
+
+        try {
+            String content = Files.lines(path).map(line -> line.replaceAll(textToReplace, replacementText)).collect(Collectors.joining("\n"));
+
+            Files.write(path, content.getBytes());
+            System.out.println("Content has been replaced in " + file.getName());
+
+        } catch (IOException e) {
+            System.err.println(e.getMessage());
+        }
+    }
+    public static String extractAissembleVersion(String line) {
```

Comment:
S: This logic makes no sense to me, as imports don't have embedded versions.  It's also not obvious that this is even being used anywhere.  Given the operation we're doing with this migration, I am skeptical we even need a Util class for this. Why can't we use the existing `FileUtils` methods for search and replace and just embed the constants of this file into the migration itself?

Reply:
I removed this function as I was not using it.